src/components/App.tsx

```diff
@@ -42,14 +47,58 @@ const App: FC<AppProps> = ({ demoId }) => {
     setEvaluatePath(true);
   }, []);
 
+  const setMcodeRecords = useCallback((resources: DomainResource[]): void => {
+    // Create a Bundle for the CQL engine
+    const bundle = {
+      resourceType: 'Bundle',
+      type: 'searchset',
+      entry: resources.map((r: DomainResource) => ({ resource: r }))
+    };
+    getFixture('mCODE.cql')
+      .then(cql => convertBasicCQL(cql))
```

Comment:
Instead of converting the CQL to ELM on every app load can we pre-convert it and store the ELM somewhere? Additionally maybe we should look into extracting the mCODE CQL library into its own repository again, though that's not part of this task

Reply:
Added pre-converted elm. I agree - the task already exists in the backlog: https://jira.mitre.org/browse/PATHWAYS-117